src/bosh-director/lib/bosh/director/task_db_writer.rb

```diff
@@ -0,0 +1,15 @@
+module Bosh::Director
+  class TaskDBWriter
+    def initialize(column, task)
+      @column_name = column
+      @task = task
+      @transactor = Transactor.new
+    end
+
+    def write(text)
+      @transactor.retryable_transaction(Bosh::Director::Config.db) do
+        @task.update({@column_name => "#{@task[@column_name]}#{text}"})
```

Comment:
@barthy1 we successfully used something like `Task.where(:id => task.id).update(description: Sequel.join([@column_name, "asdfasdf"]))`.
Refreshing the task model should not be necessary, we don't expect any code to read from the new logging columns while the task is running.